packages/fastify-vite/config.js

```diff
@@ -195,6 +196,21 @@ async function resolveBundle ({ dev, vite }) {
   return bundle
 }
 
+async function resolveSPABundle ({ dev, vite }) {
+  const bundle = {}
+  if (!dev) {
+    bundle.dir = resolve(vite.root, vite.build.outDir)
+    const indexHtmlPath = resolve(bundle.dir, 'client/index.html')
```

Comment:
I left this as /dist/client, but it may make sense to remove this hard dependency on having a /client folder. SPAs should probably just use the /dist output (or at least allow for configuration)

Any thoughts on that?

Reply:
JSYK There's no hard dependency on having a `client/` folder, `root` in `vite.config.js` can be the same folder as `server.js` and it should work fine.

Amazing work, I'll give it a go and review properly soon.

Reply:
Ok cool thanks! 

What I meant here was that AFAICT,  the output folder has to be /client right now for production. Unless there is a way of overriding how the production bundles are loaded that I missed.